platformcomponents/desktop/date-picker.json

```diff
@@ -39,22 +39,45 @@
     },
     "dayPicker":{
       "day":{
-        "background":{
-          "#normal": "@theme-button-secondary-normal",
-          "#hovered": "@theme-button-secondary-hover",
-          "#pressed": "@theme-button-secondary-pressed",
-          "#disabled": "@theme-button-secondary-normal",
-          "#active": "@theme-background-accent-normal"
+        "unchecked": {
```

Comment:
it is not a checkbox, se we can call this 'inactive' and the other one 'active'

Reply:
ok